build/teamcity/cockroach/nightlies/roachtest_nightly_impl.sh

```diff
@@ -83,6 +83,7 @@ fi
 build/teamcity-roachtest-invoke.sh \
   --metamorphic-encryption-probability=0.5 \
   --metamorphic-arm64-probability="${ARM_PROBABILITY:-0.5}" \
+  --metamorphic-fips-probability="${FIPS_PROBABILITY:-0.1}" \
```

Comment:
Because we set `arm64=0.5`, FIPS is effectively only a 5% chance of being chosen. Fine with me, just double checking if that's intended.

Reply:
Good catch! From, `P(fips | amd64) = P(fips) * (1 - P(arm64))`, and the default probabilities of arm and fips, it follows that FIPS is effectively 5%. I think I would go even lower than 5% since FIPS runs are essentially smoke tests. I'll update the default P(fips) to `0.02`.